hibernate/src/main/resources/application.yml

```diff
@@ -0,0 +1,15 @@
+spring:
```

Comment:
Do not mix different styles of property files.
If you use YAML use YAML, if *.properties - *.properties